<launch>

    <!--<node pkg="self_aiming" name="server_node" type="server_node.py" output="screen"/>-->
    <!--<node pkg="rqt_reconfigure" name="rqt_reconfigure" type="rqt_reconfigure" output="screen"/>-->

    <node name="visual_servo" pkg="gimbal_controller" type="visual_servo_controller" output="screen">
        <param name="Kp" type="double" value="-10.0"/>
        <param name="publisher_topic" type="string" value="/cmd_vel"/>
    </node>
</launch>
